.github/workflows/nightly.yaml

```diff
@@ -111,13 +111,53 @@ jobs:
         env:
           LATEST_COMMIT_SHA: ${{ github.sha }}
 
-      - name: Run SeaLights scan for e2e tests
+      - name: Run SeaLights scan for tests
         if: ${{ steps.operator-image-existence-checker.outputs.OPERATOR_IMAGE_EXISTS == 'true' }}
         run: |
-          echo "[SeaLights] Running the SeaLights e2e tests scan"
+          echo "[SeaLights] Running the SeaLights scan"
           ./slcli scan --bsid buildSessionId.txt  --path-to-scanner ./slgoagent  --workspacepath "./" --scm git --scmBaseUrl https://github.com/redhat-developer/rhdh-operator --scmProvider github
-        env:
-          SEALIGHTS_TEST_STAGE: "E2E Tests Nightly"
+
+      # gosec needs a "build" stage so connect it to the lint step which we always do
+      - name: Build
+        if: ${{ steps.operator-image-existence-checker.outputs.OPERATOR_IMAGE_EXISTS == 'true' }}
+        run: make lint
+
+      - name: Run Controller
+        if: ${{ steps.operator-image-existence-checker.outputs.OPERATOR_IMAGE_EXISTS == 'true' }}
```

Comment:
```suggestion
```

Same point, but it would require running the "Start Minikube" step as well, which should be fine.